data_rentgen/consumer/handlers.py

```diff
@@ -210,20 +210,29 @@ async def get_or_create_schema(schema: SchemaDTO, unit_of_work: UnitOfWork) -> S
 async def create_or_update_input(
     input: InputDTO,
     operation: Operation,
+    run: Run,
     dataset: Dataset,
     schema: Schema | None,
     unit_of_work: UnitOfWork,
 ) -> Input:
     schema_id = schema.id if schema else None
-    return await unit_of_work.input.create_or_update(input, operation.id, dataset.id, schema_id)
+    return await unit_of_work.input.create_or_update(
+        input,
+        operation.id,
+        run.id,
+        run.job_id,
+        dataset_id=dataset.id,
+        schema_id=schema_id,
+    )
 
 
 async def create_or_update_output(
     output: OutputDTO,
     operation: Operation,
+    run: Run,
     dataset: Dataset,
     schema: Schema | None,
     unit_of_work: UnitOfWork,
 ) -> Output:
     schema_id = schema.id if schema else None
-    return await unit_of_work.output.create_or_update(output, operation.id, dataset.id, schema_id)
+    return await unit_of_work.output.create_or_update(output, operation.id, run.id, run.job_id, dataset.id, schema_id)
```

Comment:
Please add asserts to consumer tests, to check that new attributes are populated properly

Reply:
Update `test_runs_handler_spark.py`